README.md

```diff
@@ -1,3 +1,31 @@
-# temple
+# Temple
 
-[![codecov](https://codecov.io/gh/TempleEight/temple/branch/develop/graph/badge.svg)](https://codecov.io/gh/TempleEight/temple)
\ No newline at end of file
+[![codecov](https://codecov.io/gh/TempleEight/temple/branch/develop/graph/badge.svg)](https://codecov.io/gh/TempleEight/temple)
+![](https://img.shields.io/travis/com/templeeight/temple)
```

Comment:
Should we get these to show the master version now (and make the landing page the master branch)?